src/marqo/tensor_search/api.py

```diff
@@ -1,8 +1,25 @@
 """The API entrypoint for Tensor Search"""
+
+import warnings
+from marqo.tensor_search.tensor_search_logging import get_logger
+import logging
+
+
+if get_logger(__name__).getEffectiveLevel() > logging.DEBUG:
+    # We need to suppress this warning before the dependency is imported
+
+    warnings.filterwarnings(
+        "ignore",
+        "Importing `GenerationMixin` from `src/transformers/generation_utils.py` "
```

Comment:
Should we raise a ticket with `transformers` for this? This is a warning for a 3rd party library right?

Reply:
yeah, but I suspect it is fixed by upgrading some package